monitoring/definitions/postgres.go

```diff
@@ -0,0 +1,98 @@
+package definitions
+
+import (
+	"time"
+
+	. "github.com/sourcegraph/sourcegraph/monitoring/monitoring"
+)
+
+const (
+	codeintel = "codeintel-db"
+	db        = "pgsql"
```

Comment:
variables are scoped across the entire package, so these should be a bit descriptive:

```suggestion
	dbCodeintel   = "codeintel-db"
	dbSourcegraph = "pgsql"
```

(or something like that)